docs/api-docs/cart-and-checkout/open-source-checkout.md

```diff
@@ -51,17 +51,37 @@ ReactDOM.render(
 
 You will need to host the custom checkout file online so it can be served by the store. You can use a hosting service such as [Amazon S3](https://aws.amazon.com/s3/). Using an external host will allow you the freedom to automate the build process if you wish and push updates automatically from your local machine.
 
-### Using WebDav to host a custom checkout
+### Using WebDAV to host a custom checkout
 
-You can upload a custom checkout to your store's server using WebDav, but we recommend hosting checkouts externally. See [Uploading and Linking to a File in Your Store](https://support.bigcommerce.com/s/article/How-do-I-add-and-link-to-a-file-in-my-store) for instructions to do this.
+You can upload a custom checkout to your store's server using WebDAV. The instructions to upload the `/dist` folder to the BigCommerce server using WebDav are below:
+
+1. Before proceeding ensure you have downloaded [Cyberduck](https://cyberduck.io/), our recommended WebDAV client.
+
+NOTE:  For more information on how to use Cyberduck, refer to the [File Access (WebDAV)](https://support.bigcommerce.com/s/article/File-Access-WebDAV) page.
+
+2. From your store control panel, navigate to **Server Settings** > **File Access (WebDAV)**.
+  - To automatically connect with Cyberduck, see [Single-click Login](https://support.bigcommerce.com/s/article/File-Access-WebDAV#login).
+  - To manually connect with Cyberduck, see [Connecting with Cyberduck Manually](https://support.bigcommerce.com/s/article/File-Access-WebDAV#manual).
+3. From Cyberduck, enter the `/content` folder and create a new folder named *checkout*.
+4. Navigate into the `/dist` folder in your checkout project.
+5. Copy the contents of the `/dist` folder and paste it into the checkout folder.
 
 ## Installing a custom checkout  
 
 To install a custom checkout on a store, follow these steps:
 
-1. Navigate to **Advanced Settings > Checkout Settings**
-2. Paste a link to your custom checkout in the **Script URL** field and **Save**
-3. Navigate to your live storefront to view your new custom checkout
+1. Navigate to **Advanced Settings > Checkout** in your store's control panel.
+2. Under **Checkout Type**, select **Custom Checkouts**.
+3. Type the following in the **Script URL** field, replacing <version> with the latest version number generated by running the `npm run release` command:
+  ```
+  `webdav:checkout/auto-loader-<version>.js`
+  ```
+  Prepending `webdav:` indicates that the URL is in the remote WebDAV directory. It will treat `/content` as the root WebDAV directory.
```

Comment:
```suggestion
  Prepending `webdav:` Indicates that the URL is in the remote WebDAV directory. It will treat `/content` as the root WebDAV directory.
```